matron/src/device/device_crow.c

```diff
@@ -47,10 +49,15 @@ int dev_crow_init(void *self) {
     read(d->fd, s, 255); // clear buffer
     write(d->fd,"^^i\n\0",5);    
     usleep(1000);
```

Comment:
maybe the `usleep` could move into the retry loop?

Reply:
so the usleep is to give crow time to respond.

the loop is to simply clear the entire serial buffer, so no usleep is needed.